### PR TITLE
Remove all the "active turfs at round start" warnings from Box

### PR DESCRIPTION
**Pull request by ojacobson**
Some chucklefuck had set a bunch of tiles to have `temperature = 273.15` rather than leaving the `temperature` attribute unset; I've removed the attribute from those tiles. I've also made tiles exposed to space use `../airless` subtypes more consistently, including a couple of new airless types (cult floors, magma).
- [x] Remove the hilarious Dream Maker puke
- [x] Adjacent tiles with different air, z1-6
- [x] Air tiles adjacent to space, z1-6

### CODE REVIEW

**Comment by Jordie0608** [comment]
You might want to try doing this with a text editor, Notepad++ is a good choice.

File: code/game/turfs/simulated/floor/plasteel_floor.dm [hunk 1]
```diff
@@ -122,6 +122,11 @@
 /turf/simulated/floor/plasteel/cult/narsie_act()
 	return
 
+/turf/simulated/floor/plasteel/cult/airless
+	oxygen = 0
+	nitrogen = 0
+	temperature = TCMB
+
```
Comment (ojacobson) [inline code/game/turfs/simulated/floor/plasteel_floor.dm]: haha whups

**Comment by ojacobson** [comment]
uuuuughhhh the wizards' den

It's enclosed by force walls, which aren't considered by the code that detects active atmos tiles at roundstart, but does hold in air. As a result, a bunch of the active tile warnings are from parts of the wizards' den which it thinks are exposed to space, but actually aren't. Not sure what to do here.

* Surround the wizards' den with fake space tiles and fill them with air, and enclose that in walls?
* Go the other direction and start wizards in a vacuum? (lolno)
* Replace the tiles under the force walls with walls that look like floors, so the wizards' den is actually sealed, and hope nobody notices that they can't move out of the den into space by destroying the force walls?
* Teach the active-tiles-at-round-start code (and atmos code generally) how to tell that air will not spread into a force wall?
* Redesign the wizards' den to be sealed, but still wizardy and cool?
* Ignore it? (It's cluttering up my debugging output right now.)

I'm weakly inclined to put fake-floor walls under the forcewall tiles, since it seems like the least work, but I'm not sold on it.

**Comment by optimumtact** [comment]
Fix the code that detects

**Comment by ojacobson** [comment]
sadness

**Comment by MrStonedOne** [comment]
oj, there is an object proc thats used for detecting this by the atmos code.

so override it on false walls,

**Comment by ojacobson** [comment]
@MrStonedOne Thanks, I'll dig through this and fix the detect loop.

**Comment by ojacobson** [comment]
There, I think this is good to go. I'll do the other stations in separate pull requests; doing this one was tiring.

File: code/controllers/subsystem/air.dm [hunk 2]
```diff
@@ -193,6 +193,7 @@ var/datum/subsystem/air/SSair
 					if(!T.air.check_turf_total(enemy_tile))
 						T.excited = 1
 						active_turfs |= T
+						break
```
Comment (Aranclanos) [inline code/controllers/subsystem/air.dm]: whats this

**Comment by Metacide** [comment]
@ojacobson - fixed most of MetaStation myself, so no need to work on that if you were considering it. Thanks all the same, particularly for finding the temperature var issue. Made fixing it much simpler. :+1:

File: code/controllers/subsystem/air.dm (hunk 2 above)
Comment (ojacobson) [inline code/controllers/subsystem/air.dm]: Previously, a single atmos tile floating in space would be detected four times. (This actually happened; there was a single tile of plasteel out in the middle of Z=2 somewhere.) Now it'll be detected once, consistent with the case where tiles are adjacent to other atmos tiles with differing gas mixes.

Comment (Aranclanos) [inline code/controllers/subsystem/air.dm]: yeah good find

**Pull request merged**